styles/Canonical/000-US-spellcheck.yml

```diff
@@ -0,0 +1,10 @@
+
+# Uses the built-in dictionary and filters.
+extends: spelling
+message: "The word '%s' seems to be misspelled."
+level: error
```

Comment:
It may be the blank-line or the comment at the header of this file

here's an [example workflow](https://github.com/canonical/k8s-operator/actions/runs/13620905606/job/38070238962) that worked hours before

the styles are installed from this repo